scripts/runhelpers.cake

```diff
@@ -220,7 +221,7 @@ public void DotnetPack(string outputFolder, string projectFolder, string project
 }
 
 public void DotnetPackNuspec(string outputFolder, string projectFolder, string project) {
```

Comment:
While you're here - could you rename this to "NugetPackNuspec"? It's not doing dotnet pack at all which could be very confusing